source/manual/request-fastly-tls-certificate.html.md

```diff
@@ -11,61 +11,61 @@ When transitioning HTTPS domains, these are the steps to request a TLS certifica
 
-   ![Photo of the step 4](images/fastly/3.png)
+    ![Screenshot of the Fastly website user interface on the "TLS management" tab which is nested under "Secure". There is a button in the top right labelled "+ Secure another domain".](images/fastly/3.png)
 
-5. Enter the domain name(s) that you want a TLS certificate to be created for. Select **Let's encrypt** as the certification authority. And select one of the following two TLS configurations. To decide, you need to know where your domain resolves to. A quick way to find out is:
+1. Enter the domain name(s) that you want a TLS certificate to be created for. Select **Let's encrypt** as the certification authority. And select one of the following two TLS configurations. To decide, you need to know where your domain resolves to. A quick way to find out is:
 
-   ```
-   ping <your-domain-name>
-   ```
+    ```
+    ping <your-domain-name>
+    ```
 
-   or
+    or
 
-   ```
-   dig <your-domain-name> +short
-   ```
+    ```
+    dig <your-domain-name> +short
+    ```
 
-   **bouncer.gds.map.fastly.net** - for any domain name which resolves to:
+    **bouncer.gds.map.fastly.net** - for any domain name which resolves to:
 
-   ```
-   - bouncer-cdn.production.govuk.service.gov.uk
-   - bouncer.gds.map.fastly.net
-   - 151.101.2.30, 151.101.66.30, 151.101.130.30, 151.101.194.30
-   ```
+    ```
+    - bouncer-cdn.production.govuk.service.gov.uk
+    - bouncer.gds.map.fastly.net
+    - 151.101.2.30, 151.101.66.30, 151.101.130.30, 151.101.194.30
+    ```
 
-   **www-gov-uk.map.fastly.net** - for any domain name which resolves to:
+    **www-gov-uk.map.fastly.net** - for any domain name which resolves to:
 
-   ```
-   - backend.production.alphagov.co.uk
-   - redirector-cdn.production.govuk.service.gov.uk
-   - redirector-cdn-ssl-businesslink.production.govuk.service.gov.uk
-   - redirector-cdn-ssl-directgov.production.govuk.service.gov.uk
-   - redirector-cdn-ssl-events-businesslink.production.govuk.service.gov.uk
-   - www-cdn.production.govuk.service.gov.uk
-   - www-gov-uk.map.fastly.net
-   - 151.101.0.144, 151.101.64.144, 151.101.128.144, 151.101.192.144
-   ```
+    ```
+    - backend.production.alphagov.co.uk
+    - redirector-cdn.production.govuk.service.gov.uk
+    - redirector-cdn-ssl-businesslink.production.govuk.service.gov.uk
+    - redirector-cdn-ssl-directgov.production.govuk.service.gov.uk
+    - redirector-cdn-ssl-events-businesslink.production.govuk.service.gov.uk
+    - www-cdn.production.govuk.service.gov.uk
+    - www-gov-uk.map.fastly.net
+    - 151.101.0.144, 151.101.64.144, 151.101.128.144, 151.101.192.144
+    ```
 
-   ![Photo of the step 4](images/fastly/4.png)
+    ![Screnshoot of the Fastly website user interface on the "TLS subscriptions" tab, nested under "TLS management". A form is shown where multiple domains can be added into a text area.](images/fastly/4.png)
```

Comment:
```suggestion
    ![Screnshot of the Fastly website user interface on the "TLS subscriptions" tab, nested under "TLS management". A form is shown where multiple domains can be added into a text area.](images/fastly/4.png)
```

Reply:
Oops!  The screenshot version of a photoshoot?

Reply:
oops my correction has a typo too!